// lib0/dist/decoding.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

require('./binary-1b5a1494.cjs');
require('./map-28a001c9.cjs');
require('./string-755e6b28.cjs');
require('./conditions-fb475c70.cjs');
require('./storage.cjs');
require('./environment-173b1eb9.cjs');
require('./buffer-b9594fdc.cjs');
var decoding = require('./decoding-a2a70203.cjs');



exports.Decoder = decoding.Decoder;
exports.clone = decoding.clone;
exports.createDecoder = decoding.createDecoder;
exports.hasContent = decoding.hasContent;
exports.peekUint16 = decoding.peekUint16;
exports.peekUint32 = decoding.peekUint32;
exports.peekUint8 = decoding.peekUint8;
exports.peekVarInt = decoding.peekVarInt;
exports.peekVarString = decoding.peekVarString;
exports.peekVarUint = decoding.peekVarUint;
exports.readAny = decoding.readAny;
exports.readBigInt64 = decoding.readBigInt64;
exports.readBigUint64 = decoding.readBigUint64;
exports.readFloat32 = decoding.readFloat32;
exports.readFloat64 = decoding.readFloat64;
exports.readFromDataView = decoding.readFromDataView;
exports.readTailAsUint8Array = decoding.readTailAsUint8Array;
exports.readUint16 = decoding.readUint16;
exports.readUint32 = decoding.readUint32;
exports.readUint8 = decoding.readUint8;
exports.readUint8Array = decoding.readUint8Array;
exports.readVarInt = decoding.readVarInt;
exports.readVarString = decoding.readVarString;
exports.readVarUint = decoding.readVarUint;
exports.readVarUint8Array = decoding.readVarUint8Array;
exports.skip8 = decoding.skip8;
